button.js

```diff
@@ -0,0 +1,17 @@
+//post function
+var $post = function() {
+  app.post('/articles', function(req,res){
+    var $title = $("#POST_title").val();
+    var $blog = $("#POST_blog").val();
+    console.log($title);
+    console.log($blog);
+});
+
+$('#btn').click(function() {
+  $post();
+})
+//get function
+var $get =function () {
+
+
+}
```

Comment:
Nope. We were going to actually post a blog, but we decided to use postman instead.